drv/cosmo-seq-server/src/main.rs

```diff
@@ -662,45 +760,6 @@ impl NotificationHandler for ServerImpl {
             .unwrap(); // this should be infallible
         }
 
-        // If Hubris thinks the system is up, do some basic checks
-        if matches!(self.state, PowerState::A0 | PowerState::A0PlusHP) {
-            // Detect the FPGA powering off without us
-            if state.seq != Ok(A0Sm::Done) {
-                ringbuf_entry!(Trace::UnexpectedPowerOff {
-                    our_state: self.state,
-                    seq_state: state.seq,
-                });
-                self.log_pg_registers();
-
-                // Power down to A2, updating our internal state.  We can't
-                // handle errors here, so log them and continue.
-                if let Err(e) = self
-                    .set_state_impl(PowerState::A2, StateChangeReason::Other)
-                {
-                    ringbuf_entry!(Trace::PowerDownError(e))
-                }
```

Comment:
I like the idea of dropping the polling in the sequencer entirely but I think we _might_ lose this case if we don't get a specific interrupt